Makefile

```diff
@@ -11,10 +11,22 @@ SOURCE_FILES?=$$(go list ./... | grep -v /vendor/)
 TEST_PATTERN?=.
 TEST_OPTIONS?=
 
-BIN_DIR := $(CURDIR)/bin
+BIN_DIR := ./bin
```

Comment:
I am not sure I see the difference here, other than hard coding the current directory as the reference for searching the bin directory. What is the purpose of this change?